neqo-http3/src/lib.rs

```diff
@@ -376,14 +377,10 @@ pub trait SendStream: Stream {
 }
 
 pub trait HttpSendStream: SendStream {
+    /// This function sets informational response.
```

Comment:
It sends final response header blocks also, right?  And, from what I can see, trailers and middlers (see my comment about that below).

Reply:
Yes, this is left over of the first design.